CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.6.1
+__22.03.2025__
+
+- feat: Properly handle unknown union types.
```

Comment:
I think this would count as a fix rather than a feature.
but since we don't really care about being consistent it's alright 😄